<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDockWidget" name="dock_hardware">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>152</width>
        <height>524287</height>
       </size>
      </property>
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <widget class="QWidget" name="dock_hardware_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="vl_hardware">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="cb_use_cam">
            <property name="text">
             <string>Camera </string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_use_led">
            <property name="text">
             <string>Lighting</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_autohome">
            <property name="text">
             <string>Return to home</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_use_tpt">
            <property name="text">
             <string>Use Transport</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDockWidget" name="dock_control">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tb_play_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qt_resources.qrc">
              <normaloff>:/icons/resources/ic_navigate_before_48px.svg</normaloff>:/icons/resources/ic_navigate_before_48px.svg</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_7">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qt_resources.qrc">
              <normaloff>:/icons/resources/ic_play_arrow_48px.svg</normaloff>
              <normalon>:/icons/resources/ic_pause_48px.svg</normalon>:/icons/resources/ic_play_arrow_48px.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_stop_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="qt_resources.qrc">
              <normaloff>:/icons/resources/ic_stop_48px.svg</normaloff>:/icons/resources/ic_stop_48px.svg</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_8">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qt_resources.qrc">
              <normaloff>:/icons/resources/ic_navigate_next_48px.svg</normaloff>:/icons/resources/ic_navigate_next_48px.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QDockWidget" name="dockWidget">
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <widget class="QWidget" name="dock_image">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMechascan">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="open_act"/>
    <addaction name="open_new_act"/>
    <addaction name="print_act"/>
    <addaction name="save_act"/>
    <addaction name="exit_act"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="next_act"/>
    <addaction name="prev_act"/>
    <addaction name="reload_act"/>
    <addaction name="zin_act"/>
    <addaction name="zout_act"/>
    <addaction name="rotright_act"/>
    <addaction name="rotleft_act"/>
    <addaction name="flipv_act"/>
    <addaction name="fliph_act"/>
    <addaction name="fulls_act"/>
    <addaction name="fit_win_act"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="help_act"/>
    <addaction name="about_act"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="crop_act"/>
    <addaction name="resize_act"/>
    <addaction name="prefs_act "/>
    <addaction name="props_act"/>
   </widget>
   <addaction name="menuMechascan"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_act">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="rotright_act">
   <property name="text">
    <string>Rotate right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="fliph_act">
   <property name="text">
    <string>Flip image horizontally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="next_act">
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="prev_act">
   <property name="text">
    <string>Previous image</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="rotleft_act">
   <property name="text">
    <string>Rotate left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="flipv_act">
   <property name="text">
    <string>Flip image vertically</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="zin_act">
   <property name="icon">
    <iconset theme="edrf">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="zout_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="open_new_act">
   <property name="text">
    <string>Open new window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="reload_act">
   <property name="text">
    <string>&amp;Reload image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="print_act">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="save_act">
   <property name="text">
    <string>&amp;Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="close_act">
   <property name="text">
    <string>Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="exit_act">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="fulls_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="ss_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Slideshow</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="ss_next_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Next / Random image</string>
   </property>
  </action>
  <action name="resize_act">
   <property name="text">
    <string>Resize image</string>
   </property>
  </action>
  <action name="crop_act">
   <property name="text">
    <string>Crop image</string>
   </property>
  </action>
  <action name="about_act">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="zout">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="fit_win_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Best &amp;fit</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="prefs_act ">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="props_act">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="help_act">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>te</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
